extensions/wikia/PageShare/PageShareHelper.class.php

```diff
@@ -17,6 +20,29 @@ public static function getIcon( $service ) {
 		return $icon;
 	}
 
+	/**
+	 * Get language for Page Share service.
+	 * For anon users use the browser language, if header is empty default to English.
+	 * For logged in user use user's language.
+	 * Both values can be overwritten by ?uselang parameter.
+	 *
+	 * @return String language
+	 */
+	public static function getLangForPageShare() {
+		global $wgLang, $wgRequest, $wgUser;
+
+		if ( $wgUser->isAnon() ) {
+			if ( array_key_exists( 'HTTP_ACCEPT_LANGUAGE', $_SERVER ) ) {
```

Comment:
I checked on my machine in the command line client (`PHP 5.5.9-1ubuntu4.7`) and what is crucial `isset($arr['key'])` returns `true` even if the `'key'` has a `false`-y value (`''`, `0`, `[]` etc., **but except** `NULL`) as is expected. Whereas `empty($arr['key'])` would return `true` for non-existing `'key'`, `'key'` with empty value and if `'key'` was `NULL`. I'd suggest using `!empty(…)` here.